README.md

```diff
@@ -273,10 +273,11 @@ TOC is generated by [markdown-toc](https://github.com/jonschlinkert/markdown-toc
 ## Compilers and Interpreters
 
 - [Aro](https://github.com/Vexu/arocc) : Aro. A C compiler with the goal of providing fast compilation and low memory usage with good diagnostics.
-- [Vexu/bog](https://github.com/Vexu/bog) : Small, strongly typed, embeddable language.
-- [squeek502/zua](https://github.com/squeek502/zua) : An implementation of Lua 5.1 in Zig, for learning purposes.
 - [buzz](https://github.com/buzz-language/buzz): A small/lightweight statically typed scripting language.
 - [fubark/cyber](https://github.com/fubark/cyber) : Fast and concurrent scripting.
+- [shepherdjerred/macos-cross-compiler](https://github.com/shepherdjerred/macos-cross-compiler) : Compile binaries for macOS on Linux.
```

Comment:
It's weird to to put it under this section, maybe `Others`?

Reply:
sounds fine to me. :)